src/SDKs/ContainerRegistry/dataplane/Microsoft.Azure.ContainerRegistry/Generated/SdkInfo_AzureContainerRegistry.cs

```diff
@@ -0,0 +1,41 @@
+
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.ContainerRegistry
+{
+  using System;
+  using System.Collections.Generic;
+  using System.Linq;
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_AzureContainerRegistry
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("AzureContainerRegistry", "DeleteAcrRepository", "2018-08-10-preview"),
```

Comment:
Please run `msbuild build.proj /p:Scope=SDKs\ContainerRegistry\dataplane` and commit the changes made to .props

Reply:
I didn't see any change in any .props file after the build. What are you expecting?

Reply:
Simply add `2018-08-10-preview;` to `AzureApiTag`